examples/util/openvino/OpenVINOModelImporter.js

```diff
@@ -237,12 +247,17 @@ class OpenVINOModelImporter {
   _getTypeCode(dataType) {
     let type;
     switch (dataType) {
+      case '?':
```

Comment:
Any reason to add "?" case?

Reply:
> Add "?" case and nothing to do?

Some ops' input type named "?" but actually represents "float32". I will remove it and modify some codes instead, thanks!